File: 2089-find-target-indices-after-sorting-array/2089-find-target-indices-after-sorting-array.java
class Solution {
    public List<Integer> targetIndices(int[] nums, int target) {
        ArrayList<Integer> list = new ArrayList<>();
        Arrays.sort(nums);
        for(int i=0;i<nums.length;i++){
            if(nums[i] == target){
                list.add(i);
            }
        }
        return list;
    }
}

File: 2089-find-target-indices-after-sorting-array/2089-find-target-indices-after-sorting-array.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var targetIndices = function(nums, target) {
    nums.sort((a,b) => a-b);
    let arr = [];
    for(let i = 0;i<nums.length;i++){
        if(nums[i] == target){
            arr.push(i);
        }
    }
    return arr;
};